packages/vite/src/executors/dev/executor.ts

```diff
@@ -35,6 +37,17 @@ export default async function runExecutor(
     context.configurationName
   );
 
+  // look for the proxy.conf.json
+  let proxyConfig: Record<string, string | ProxyOptions>;
+  const proxyConfigPath = options.proxyConfig
+    ? join(context.root, options.proxyConfig)
+    : join(projectRoot, 'proxy.conf.json');
+
+  if (existsSync(proxyConfigPath)) {
+    console.info(`found proxy configuration at ${proxyConfigPath}`);
```

Comment:
Better use logger from @nrwl/devkit with logger.info()

Reply:
Resolved. Not sure if we should merge this now. Or wait for generator to be finished (hoping some point tonight) and have migration ready for svelte

Reply:
I don’t see a reason not to merge. But probably add some information and maybe the example proxy conf to the docs would be a good idea before we do the release